### PR TITLE
[16.0][FIX] stock_account_product_run_fifo_hook, rewrite to original method

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Update method to keep original algorithm in current version

@ForgeFlow
cc. @JordiBForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — COMMENTED** [review]
Could you add a comment like `# START OF CHANGES` and `# END OF CHANGES` to clearly depict what is being modified in the standard method?

File: stock_account_product_run_fifo_hook/hooks.py [hunk 1]
```diff
@@ -122,157 +125,181 @@ def _run_fifo_new(self, quantity, company):
...
+                    new_remaining_value = (
+                        candidate.remaining_value - value_taken_on_candidate
+                    )
+
+                    candidate_vals = {
+                        "remaining_qty": candidate.remaining_qty
+                        - qty_taken_on_candidate,
+                        "remaining_value": new_remaining_value,
+                    }
+                    # Start Hook
```
Comment (ChrisOForgeFlow) [inline stock_account_product_run_fifo_hook/hooks.py:222]: @LoisRForgeFlow Here we add a function helper to allow change data to write in candidates

Comment (LoisRForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py:222]: Perfect, that's what I missed

Thanks!

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Pull request merged**